### PR TITLE
Add more common messages for future PR's

### PR DESCRIPTION
**Pull request by Mysterious-Dev**
All is in the title ^^.

### CODE REVIEW

**Review by triphora — APPROVED** [review]

**Comment by Mysterious-Dev** [comment]
@triphora @Geometrically  I added some new keys.

File: utils/common-messages.ts [hunk 1]
```diff
@@ -7,6 +7,14 @@ export const commonMessages = defineMessages({
...
     defaultMessage: 'Cancel',
   },
+  continueButton: {
+    id: 'button.continue',
+    defaultMessage: 'Continue',
+  },
+  createdAtLabel: {
+    id: 'label.created-at',
+    defaultMessage: 'Created {ago}',
+  },
```
Comment (brawaru) [inline utils/common-messages.ts:17]: Here message content does not match the ID and property name.
```suggestion
  createdAgoLabel: {
    id: 'label.created-ago',
    defaultMessage: 'Created {ago}',
  },
```

Comment (Mysterious-Dev) [reply utils/common-messages.ts]: Yeah, that's better ^^.

**Comment by brawaru** [comment]
Don't forget to extract messages after updating message definitions 👍🏻

**Comment by Mysterious-Dev** [comment]
> Don't forget to extract messages after updating message definitions 👍🏻

That's fixed ^^

**Comment by brawaru** [comment]
What happened here

**Comment by brawaru** [comment]
I have rebased and squashed this branch, you can add my remote and reset this branch to one of my branches (but please read my message at the bottom first):
```bash
git remote add brawaru https://github.com/brawaru/modrinth-knossos.git
git fetch --all

# (Optionally) Make backup of this branch state:
git checkout -b backup/more-common-messages more-common-messages

git checkout more-common-message

# Reset to non-squashed rebase:
git reset --hard brawaru/feat/more-common-messages
# or Reset to a squashed rebase:
git reset --hard brawaru/squashed/feat/more-common-messages

# Override your branch state
git push --force-with-lease
```

I'd really really recommend you squash your changes and use rebase instead of merge so you don't find yourself in these annoying situations.

That said, I don't think this PR makes a lot of sense on its own. It contains two unrelated and dangling changes. I'd say, keep it closed and don't commonise messages until all your other PRs are merged: just make duplicate messages; having, say, `profile.button.report` and `project.button.report`, is fine. We can commonise messages to reduce duplicates later, when we're done localising a bigger chunk of pages.

**Pull request closed without merging**